flutter-idea/src/io/flutter/editor/FlutterCompletionContributor.java

```diff
@@ -11,7 +11,7 @@
 import com.intellij.util.ui.EmptyIcon;
 import com.jetbrains.lang.dart.ide.completion.DartCompletionExtension;
 import com.jetbrains.lang.dart.ide.completion.DartServerCompletionContributor;
-import org.apache.commons.lang.StringUtils;
```

Comment:
As a drive by comment, perhaps add this to the discouraged imports here: https://github.com/flutter/flutter-intellij/blob/master/tool/plugin/lib/lint.dart#L83.

Reply:
Yes, done.  Without modifying it the PR didn't pass.

Thanks Devon.